exercises/numbers/power/power.js

```diff
@@ -11,6 +11,13 @@
  * @returns {number} The base raised to the power of the exopnent
  */
 function power(base, exponent) {
+  if (exponent === 0){
+    return 1
+  }
+  while ( exponent > 1) {
```

Comment:
There's no reason to have a `while` loop here. If you enter the loop body you return immediately, so functionally this is equivalent to `if (exponent > 1) { ... }`.

But why check if `exponent > 1` at all?

If the function is meant to handle negative exponents, you'd want to also handle that case. If the function is _not_ meant to handle negative exponents (e.g., you're assuming that `exponent >= 0` already) then if `exponent !== 0` you automatically know that `exponent > 1`.

Again, think about how you'd do this by hand and start there.

If you wanted to do something recursive, it'd go like this:

```js
function power(base, exponent) {
  if (exponent === 0) {
    return 1;
  }

  if (exponent < 0) {
    return 1 / power(base, -1 * exponent);
  }

  return base * power(base, exponent - 1);
}
```